lib/ws.js

```diff
@@ -69,6 +72,35 @@ class BinanceWS {
         return data;
     }
 
+    _clearUserDataInterval() {
+        this.userDataRefreshIntervalId &&
+            clearInterval(this.userDataRefreshIntervalId);
+        this.userDataRefreshIntervalId = false;
+    }
+
+    _sendUserDataKeepAlive(binanceRest, response) {
+        return binanceRest.keepAliveUserDataStream(response).catch(e => {
+            this.userDataRefreshIntervalId.failCount++;
+            const msg =
+                'Failed requesting keepAliveUserDataStream for onUserData listener';
+            if (e && e.code === -1125) {
+                console.error(
+                    new Date(),
+                    msg + ' listen key expired - clearing keepAlive interval',
+                    e
+                );
+                return this._clearUserDataInterval();
```

Comment:
Why are you returning this?  It looks like `this._clearUserDataInterval` just returns `undefined`?  Why not just `return`?

Reply:
Idea was two actions in one line, instead of two lines to execute the clear then early return. I can change it to this if preferred:
```
                this._clearUserDataInterval();
                return; 
```

Reply:
I usually try and optimize for readability and understanding when possible, and let V8 handle optimizing the little things that affect performance.